make_efi_entries

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/bash
+
+if [[ ! -f /sys/firmware/efi/ ]]; then
+    exit 0
+fi
+
+ARCH=$(uname -m)
+AR=x64
+if [[ $arch = aarch64 ]]; then
+    AR=aa64;
+fi
+
+if [[ -f /etc/os-release ]]; then
+    DISTRO=$(grep "^ID=" /etc/os-release | sed -e 's/ID=//' -e 's/\"//g')
+fi
+
+if [[ -f /etc/kernel/cmdline ]]; then
+    BOOT_OPTIONS="$(tr -s ' ' < /etc/kernel/cmdline)"
+else
+    BOOT_OPTIONS="$(tr -s '\n' < /proc/cmdline | sed -e 's/BOOT_IMAGE=[^ ]*//')"
```

Comment:
I think this would lead a leading space

$ tr -s '\n' < /proc/cmdline | sed -e 's/BOOT_IMAGE=[^ ]*//'
 root=UUID=1ed1e73c-5aa5-4b08-a28e-c746a49b8cb7 ro rootflags=subvol=root rd.luks.allow-discards rd.luks.options=discard rd.luks.uuid=luks-c6bc9e8f-acd6-4a25-8f26-051a35329bee rhgb quiet rd.luks.key=c6bc9e8f-acd6-4a25-8f26-051a35329bee=/keyfile:UUID=817d7a19-a8ad-45a2-a918-7ba49deb0cbd